File: VB/AdditionalCommands.Module.Web/WebModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel

Imports DevExpress.ExpressApp

Namespace AdditionalCommands.Module.Web
	<ToolboxItemFilter("Xaf.Platform.Web")> _
	Public NotInheritable Partial Class AdditionalCommandsAspNetModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
	End Class
End Namespace

File: CS/AdditionalCommands.Module.Web/WebModule.cs
using System;
using System.ComponentModel;

using DevExpress.ExpressApp;

namespace AdditionalCommands.Module.Web
{
    [ToolboxItemFilter("Xaf.Platform.Web")]
    public sealed partial class AdditionalCommandsAspNetModule : ModuleBase
    {
        public AdditionalCommandsAspNetModule()
        {
            InitializeComponent();
        }
    }
}
